ProcesoCierreIteración.md

```diff
@@ -1,11 +1,12 @@
 # Proceso de Cierre de Iteración
-Versión 1.0
+Versión 1.2
 
 
 []() | []()
 --|--
 Objetivo| Concluir la iteración
-Métricas utilizadas | Velocidad del equipo en porcentaje del proyecto y el porcentaje que representa la iteración hacia el proyecto en general.
+Métricas utilizadas | Número de patrones y antipatrones presentes en la iteración
+Repositorio de Métricas | Tabla de patrones y antipatrones en el documento de cierre de iteración "Bob the Builder" en la wiki de cada equipo. Actualizado cada fin de iteración por el team leader.
```

Comment:
Creo que es recomendable que o pongas en negritas "Bob the Builder" o pongas ahí el link. Como es la primera vez que se menciona, no queda claro qué es, a menos de que veas el documento